src/afa/afa.cc

```diff
@@ -1121,6 +1115,24 @@ Afa Mata::Afa::construct(
                       create_node(curr_graph->collect_node_names()));
     }
 
+	// TODO final states can be also given as true/false
+	if (inter_aut.are_final_states_conjunction_of_negation()) {
+		// final states are given as a conjunction of non-final states
```

Comment:
Maybe you can encapsulate this to intermediate automaton class to make code here cleaner.

Reply:
This was just a quick fix so it parses final states correctly (and there is still the need to fix true/false). To make it a method of intermediate automaton, I would need to get states from the transition relation and I am not sure how exactly it looks for AFA.